guides/common/modules/proc_importing-container-images.adoc

```diff
@@ -22,7 +22,7 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-importing-contain
 . In the *Registry Search Parameter* field, enter any search criteria that you want to use to filter your search, and then click *Discover*.
 . Optional: To further refine the *Discovered Repository* list, in the *Filter* field, enter any additional search criteria that you want to use.
 . From the *Discovered Repository* list, select any repositories that you want to import, and then click *Create Selected*.
-. Optional: To change the download policy for this docker repository to _on demand_, see xref:changing_the_download_policy_for_a_repository_{context}[].
+. Optional: To change the download policy for this Docker repository to _on demand_, see xref:changing_the_download_policy_for_a_repository_{context}[].
```

Comment:
Maybe also go with "container repository" as above in the other file? Your call.